manifests/config.pp

```diff
@@ -247,12 +139,12 @@
       File["${conf_dir}/conf.d"] {
         purge   => true,
         recurse => true,
-        notify  => Class['::nginx::service'],
+        notify  => Class['service'],
```

Comment:
That is a  _good_ question. To be honest, I think that's the result of an overzealous sed command or search / replace!

Updated and pushed, thanks for catching that.